### PR TITLE
Update Dependency-Resolution.md

### PR DESCRIPTION
**Pull request by nkolev92**
Fixes https://github.com/NuGet/docs.microsoft.com-nuget/issues/852. 

Not sure how we are supposed to link docs like https://docs.microsoft.com/en-us/visualstudio/msbuild/common-msbuild-project-properties?view=vs-2017#list-of-common-properties-and-parameters 

Use the full link? or?
/cc @karann-msft @anangaur @rrelyea ?

### CODE REVIEW

**Comment by karann-msft** [comment]
OPS Build status updates of commit _[fae6c99](https://github.com/NuGet/docs.microsoft.com-nuget/pull/1284/commits/fae6c994c4194e8103b7b1ba39d9812858f1ed9c)_: 

:clock10: **Preparing**: average preparing time is _40 sec(s)_

**Comment by karann-msft** [comment]
OPS Build status updates of commit _[fae6c99](https://github.com/NuGet/docs.microsoft.com-nuget/pull/1284/commits/fae6c994c4194e8103b7b1ba39d9812858f1ed9c)_: 

:clock10: **Incremental building**: average incremental building time is _21 sec(s)_

**Comment by karann-msft** [comment]
OPS Build status updates of commit _[fae6c99](https://github.com/NuGet/docs.microsoft.com-nuget/pull/1284/commits/fae6c994c4194e8103b7b1ba39d9812858f1ed9c)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/consume-packages/Dependency-Resolution.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/nkolev92-projectassetsjson/docs/consume-packages/Dependency-Resolution.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/consume-packages/dependency-resolution?branch=pr-en-us-1284) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C1%5C25%5C19a7edc0-f91d-0fd5-6e5f-4d131db05c11%5CPullRequest%5C201901251908592577-1284%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=meZnsw2XZS%2Busd%2FzctjdDvDTzpc%2BqzQYbAquYNWurGU%3D&st=2019-01-25T19%3A05%3A02Z&se=2019-02-25T19%3A10%3A02Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by karann-msft** [comment]
OPS Build status updates of commit _[70f9448](https://github.com/NuGet/docs.microsoft.com-nuget/pull/1284/commits/70f94484ed5033f94d978a2330487addb39d522e)_: 

:clock10: **Preparing**: average preparing time is _40 sec(s)_

**Comment by karann-msft** [comment]
OPS Build status updates of commit _[70f9448](https://github.com/NuGet/docs.microsoft.com-nuget/pull/1284/commits/70f94484ed5033f94d978a2330487addb39d522e)_: 

:clock10: **Incremental building**: average incremental building time is _21 sec(s)_

**Comment by karann-msft** [comment]
OPS Build status updates of commit _[70f9448](https://github.com/NuGet/docs.microsoft.com-nuget/pull/1284/commits/70f94484ed5033f94d978a2330487addb39d522e)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/consume-packages/Dependency-Resolution.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/nkolev92-projectassetsjson/docs/consume-packages/Dependency-Resolution.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/consume-packages/dependency-resolution?branch=pr-en-us-1284) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C1%5C25%5C19a7edc0-f91d-0fd5-6e5f-4d131db05c11%5CPullRequest%5C201901251909470372-1284%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=fQxk64IEcky1kea5d7IHNQWrzzGh3C926v6u15xZoZc%3D&st=2019-01-25T19%3A06%3A06Z&se=2019-02-25T19%3A11%3A06Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/consume-packages/Dependency-Resolution.md [hunk 1]
```diff
@@ -19,7 +19,9 @@ When multiple packages have the same dependency, then the same package ID can ap
...
 
-When the NuGet restore process runs prior to a build, it resolves dependencies first in memory, then writes the resulting graph to a file called `project.assets.json` in the `obj` folder of a project using PackageReference. MSBuild then reads this file and translates it into a set of folders where potential references can be found, and then adds them to the project tree in memory.
+When the NuGet restore process runs prior to a build, it resolves dependencies first in memory, then writes the resulting graph to a file called `project.assets.json`. 
+The assets file is located at `MSBuildProjectExtensionsPath`, which defaults to the 'obj' of the project.
```
Comment (loic-sharma) [inline docs/consume-packages/Dependency-Resolution.md]: `which defaults to the 'obj' of the project` -> `which defaults to the project's 'obj' folder`?

Comment (loic-sharma) [reply docs/consume-packages/Dependency-Resolution.md]: It's not super clear that `MSBuildProjectExtensionsPath` is an MSBuild variable. Could this be clarified?

Comment (nkolev92) [reply docs/consume-packages/Dependency-Resolution.md]: The MSBuild part wasn't clear enough? :D

Regardless, I want to link the known msbuild properties page like https://docs.microsoft.com/en-us/visualstudio/msbuild/common-msbuild-project-properties?view=vs-2017#list-of-common-properties-and-parameters, but I'm not 100% sure how we usually link to page like that (do we use the full link etc).

**Comment by karann-msft** [comment]
@nkolev92 has all the feedback been incorporated? and is the PR good to be merged?

**Comment by karann-msft** [comment]
OPS Build status updates of commit _[36a9595](https://github.com/NuGet/docs.microsoft.com-nuget/pull/1284/commits/36a95951e8f0cd0711a562cb4328342465fbd03f)_: 

:clock10: **Preparing**: average preparing time is _40 sec(s)_

**Comment by nkolev92** [comment]
@karann-msft Fixed it now.
Can be merged.

**Comment by karann-msft** [comment]
OPS Build status updates of commit _[36a9595](https://github.com/NuGet/docs.microsoft.com-nuget/pull/1284/commits/36a95951e8f0cd0711a562cb4328342465fbd03f)_: 

:clock10: **Incremental building**: average incremental building time is _21 sec(s)_

**Comment by karann-msft** [comment]
OPS Build status updates of commit _[36a9595](https://github.com/NuGet/docs.microsoft.com-nuget/pull/1284/commits/36a95951e8f0cd0711a562cb4328342465fbd03f)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/consume-packages/Dependency-Resolution.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/nkolev92-projectassetsjson/docs/consume-packages/Dependency-Resolution.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/consume-packages/dependency-resolution?branch=pr-en-us-1284) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C2%5C13%5C19a7edc0-f91d-0fd5-6e5f-4d131db05c11%5CPullRequest%5C201902131954040901-1284%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=VSpi0Me9iyaQn%2B70YpHkiKFuo2i69p%2BtJ5oLiK9ENAg%3D&st=2019-02-13T19%3A50%3A09Z&se=2019-03-16T19%3A55%3A09Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by zivkan** [comment]
> Not sure how we are supposed to link docs like https://docs.microsoft.com/en-us/visualstudio/msbuild/common-msbuild-project-properties?view=vs-2017#list-of-common-properties-and-parameters
> 
> Use the full link? or?

It's a bit late for this PR, but I found how we're supposed to do it: https://review.docs.microsoft.com/en-us/help/contribute/links-how-to?branch=master#links-to-articles-in-other-repos-or-docsets

**Comment by karann-msft** [comment]
@zivkan its not late since it isn't live yet :)

**Pull request merged**